modules/custom/d_p_side_image/templates/paragraph--d-p-side-image.html.twig

```diff
@@ -73,7 +73,12 @@
         <div class="container">
           <div class="row">
             <div class="{{ text_class }} container-half">
-              {{ content.group_d_side_image_content }}
+              {{ content.group_d_side_image_content | without('field_d_cta_link') }}
```

Comment:
Tu jest troche bezsens. Wyciągnij w takim raize na zarządzaniu wyświetlaniem pole z linkiem poza grupę.
Niestety będziesz musiał napisać update, ale ma to wiekszy sens niz wyciagnie w twigu z grupy czegos, co i tak do tej grupy nie ma nalezec.

Reply:
Jeśli chodzi o pisanie update to na razie mamy zamrożoną ścieżkę migracji, jak coś to może wpaść do kolejnej wersji Drooplera.

Reply:
Pytanie też, czy wyciągnięcie pola za grupę nie spowoduje zniknięcia go z arrayki przekazanej do Twiga

Reply:
@grzegorz-pietrzak-droptica czyli tego nie ruszać teraz, a zrobimy taska na następną wersją by takie przypadki robić?

Reply:
jak nie ma migracji, to trzeba by tak zrobić
bo sami przyznacie ze wkładać na display mode pole w grupe, poto zeby ja jednak wyciagac w twigu to sredni pomysl?

Reply:
Chyba ze zamkniemy ten ticket w ten sposób jak powyżej, a do kolejnej wersji dodamy ticket żeby poprawić. Bo tak znowu zostanie jakaś niespójność niedokończona.

Reply:
Tak no to ma sens, trzeba by załozyc ticket na wyciagniecie tego z grupy i tyle :-)